site/assets/scss/_scrolling.scss

```diff
@@ -0,0 +1,5 @@
+// When navigating with the keyboard, prevent focus from landing behind the sticky header
+
+main *:focus {
+  scroll-margin-top: 100px;
```

Comment:
**[Bootstrap Side]** Doesn't seem to ork on my side + scroll-margin-bottom if it works to avoid being at the bottom edge ?

Reply:
See https://github.com/twbs/bootstrap/pull/38199